task/06-conditions-n-loops-tasks.js

```diff
@@ -431,7 +561,71 @@ function getMatrixProduct(m1, m2) {
  *
  */
 function evaluateTicTacToePosition(position) {
-    throw new Error('Not implemented');
+    let counterX;
+    let counter0;
+    for (let i = 0; i < position.length; i++) {
+        counter0 = 0;
+        counterX = 0;
+        for (let j = 0; j < position[i].length; j++) {
+            if (position[i][j] == 'X'){
+                counterX = counterX + 1;
+            }else if (position[i][j] == '0'){
+                counter0 = counter0 + 1;
+            }
+        }
+        if (counterX == 3){
+            return 'X';
+        }else if (counter0 == 3){
+            return '0';
+        }
+    }
+    for (let i = 0; i < position.length; i++) {
+        counter0 = 0;
+        counterX = 0;
+        for (let j = 0; j < position[i].length; j++) {
+            if (position[j][i] == 'X'){
+                counterX = counterX + 1;
+            }else if (position[j][i] == '0'){
+                counter0 = counter0 + 1;
+            }
+        }
+        if (counterX == 3){
+            return 'X';
+        }else if (counter0 == 3){
+            return '0';
+        }
+    }
+    counter0 = 0;
+    counterX = 0;
+    for (let i = 0; i < position.length; i++) {
+        if (position[i][i] == 'X'){
+            counterX = counterX + 1;
+        }else if (position[i][i] == '0'){
+            counter0 = counter0 + 1;
+        }
+        if (counterX == 3){
+            return 'X';
+        }else if (counter0 == 3){
```

Comment:
You must use google's js code style guid

see the 2nd point under 4.6.2
https://google.github.io/styleguide/jsguide.html#formatting-horizontal-whitespace